Apps/Src/SendTritium.c

```diff
@@ -155,7 +164,7 @@ static float percentToFloat(uint8_t percent){
 /**
  * @brief Dumps info to UART during testing
 */
-#ifdef __TEST_SENDTRITIUM
+#ifdef DEBUG
```

Comment:
Can we move all this stuff to the test file as well? Since we are creating getters for the variables anyways, we should be able to implement this in the test file.

Reply:
These debug functions are called in the FSM sequence (it's in the task's while loop). They are not called in the test file. So we can't move them, since they would do nothing in the test file (and cause warnings). It dumps debug info every FSM transition. I guess the real question is if you want to have a a debugging instrument within the FSM code. I think it is fine as is. I don't think having debugging instruments in our code is a bad thing, as long as we macro them out, such as here

Reply:
I think it's fine for M2 but I'd eventually like to isolate all test/debug instruments to the test files specifically. That's more of a long-term goal though.